schema-engine/core/src/state.rs

```diff
@@ -41,6 +41,7 @@ type ErasedConnectorRequest = Box<
 >;
 
 impl EngineState {
+    /// TODO
```

Comment:
Is there anything to do here? If you've changed your mind on that TODO, please remove the comment

Reply:
Ah yeah, it was because I needed to expose that when importing the schema core. It is gone.